src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -66,7 +78,16 @@ class TensorRTRuntime : public JSONRuntimeBase {
         use_implicit_batch_(true),
         max_workspace_size_(size_t(1) << 30),
         max_batch_size_(-1),
-        multi_engine_mode_(false) {}
+        multi_engine_mode_(false) {
+          const bool use_int8 = dmlc::GetEnv("TVM_TENSORRT_USE_INT8", false);
```

Comment:
Suggest to move these options to `LoadGlobalAttributes` and remove the use of environment variables(also for the FP16).

Reply:
Ok, thank you for your suggestion, and I will check them and modify it